<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_NewPiece</class>
 <widget class="QWidget" name="Form_NewPiece">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>9</y>
     <width>511</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fabricant</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_Manufacturer"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_NewManufacturer">
      <property name="text">
       <string>Nouveau fabricant</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_PieceName"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Numéro EAN</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_EANPiece"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPlainTextEdit" name="plainTextEdit_DescriptionPiece"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Prix unitaire</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_PrixPiece"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Véhicule</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBox_Vehicule"/>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_NewVehicule">
      <property name="text">
       <string>Nouveau véhicule</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_AjouterPiece">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>532</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter pièce</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_Manufacturer</tabstop>
  <tabstop>pushButton_NewManufacturer</tabstop>
  <tabstop>lineEdit_PieceName</tabstop>
  <tabstop>lineEdit_EANPiece</tabstop>
  <tabstop>plainTextEdit_DescriptionPiece</tabstop>
  <tabstop>doubleSpinBox_PrixPiece</tabstop>
  <tabstop>comboBox_Vehicule</tabstop>
  <tabstop>pushButton_NewVehicule</tabstop>
  <tabstop>pushButton_AjouterPiece</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
